version-specific/mqt-compat/.pylintrc.jinja

```diff
@@ -0,0 +1,100 @@
+[MASTER]
```

Comment:
It'd probably be nice to use the same `{% extends "src/.pylintrc-mandatory.jinja" %}` mechanism here as in `src/`.

Reply:
Proposal in https://github.com/coopiteasy/oca-addons-repo-template/commit/9ef6115740c3b8514d270cc33118de6f1613ab30

Reply:
I've no strong opinion on this. As long as this does the same as the [pylint configs of oca/maintainer-quality-tools](https://github.com/OCA/maintainer-quality-tools/tree/master/travis/cfg).

Reply:
It does basically the same, except `.pylintrc` now also repeats the `.pylintrc-mandatory` tests, which I think is desirable for local development.